CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
```

Comment:
Is this code of conduct something we can legally commit Expel to? Might want to vet this.